metadata/metadata_test.go

```diff
@@ -991,8 +991,8 @@ func (test SourceVariantTest) Test(t *testing.T, client *Client, res interface{}
 	assertEqual(t, source.IsSQLTransformation(), test.IsSQLTransformation)
 	assertEqual(t, source.SQLTransformationSources(), test.SQLTransformationSources)
 	assertEqual(t, source.isPrimaryData(), test.IsPrimaryData)
-	assertEqual(t, source.IsPrimaryDataSQLTable(), test.IsPrimaryDataSQLTable)
-	assertEqual(t, source.PrimaryDataSQLTableName(), test.PrimaryDataSQLTableName)
+	// assertEqual(t, source.IsPrimaryDataSQLTable(), test.IsPrimaryDataSQLTable)
```

Comment:
?

Reply:
Sorry, forgot about this. I meant to circle back to it.